selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java

```diff
@@ -109,6 +111,7 @@ public String getTextFromFindUsagesPanel() {
   public void clickOnIconNodeInFindUsagesPanel(String nameNode) {
     seleniumWebDriverHelper.waitAndClick(
         By.xpath(format(Locators.FIND_USAGES_ICON_NODE, nameNode)));
+    WaitUtils.sleepQuietly(500, TimeUnit.MILLISECONDS);
```

Comment:
Could you, please, leave a comment in the code with explanation why do we need to wait here for 500 millisecs?